interface/GoodParticle.h

```diff
@@ -0,0 +1,28 @@
+#include <TLorentzVector.h>
```

Comment:
Needs and include guard or it will cause problems if it's included in multiple selectors. You can see I added this for the BranchManager.h: https://github.com/kdlong/VVAnalysis/blob/master/interface/BranchManager.h#L1-L2

Reply:
Yah, I should probably add that... I'll change that up